### PR TITLE
Consolidating serialization tests.

### PR DESCRIPTION
**Pull request by SGuyGe**

### CODE REVIEW

**Comment by JonHanna** [comment]
Do people really find these forms easier to read?

File: src/System.Runtime.Serialization.Json/tests/DataContractJsonSerializer.cs [hunk 1]
```diff
@@ -1324,21 +1670,43 @@ private static T SerializeAndDeserialize<T>(T value, string baseline, DataContra
             dcjs = (settings != null) ? new DataContractJsonSerializer(typeof(T), settings) : new DataContractJsonSerializer(typeof(T));
         }
 
+        Console.WriteLine("Testing input value : {0}", value);
```
Comment (JonHanna) [inline src/System.Runtime.Serialization.Json/tests/DataContractJsonSerializer.cs]: Why is this necessary?

**Comment by SGuyGe** [comment]
@JonHanna Currently we have different styles mixed together, which is the worst situation to be in. Many people find it a lot easier to read, especially for multiline strings. Of course, I don't think it's something that everyone would agree on, because different people may have different personal preferences. Regardless, it's better to be consistent than ad hoc.

**Comment by stephentoub** [comment]
Compilation failure:
```
CSC : error CS2001: Source file 'd:\j\workspace\dotnet_corefx_windows_debug_prtest\src\Common\tests\System\Console.cs' could not be found [d:\j\workspace\dotnet_corefx_windows_debug_prtest\src\System.Runtime.Serialization.Json\tests\System.Runtime.Serialization.Json.Tests.csproj]
```
We shouldn't be using Console.WriteLine in tests, and that test Console.cs (which was just a wrapper for Debug.WriteLine) was removed a few weeks ago.

**Comment by SGuyGe** [comment]
@stephentoub Thanks Stephen. This should be a merge problem, let me fix that.

File: src/System.Runtime.Serialization.Json/tests/DataContractJsonSerializer.cs (hunk 1 above)
Comment (SGuyGe) [inline src/System.Runtime.Serialization.Json/tests/DataContractJsonSerializer.cs]: This should be for debugging purpose and has existed for a while. Now it should be changed to Debug.WriteLine. Apparently I've got a merge issue. I'll fix that.

**Comment by SGuyGe** [comment]
@shmao @zhenlan @khdang

**Comment by SGuyGe** [comment]
Will do a rebase for a clean history.
